robyn/__init__.py

```diff
@@ -31,8 +31,9 @@ def __init__(self, file_object: str, config: Config = Config()) -> None:
         directory_path = os.path.dirname(os.path.abspath(file_object))
         self.file_path = file_object
         self.directory_path = directory_path
+        self.testName = ""
         self.config = config
-
+        self.dependencies = {"all":{'request':Request, 'response':Response}} #new
```

Comment:
Can you please explain the logic behind the structure here?

Reply:
In inner_handler(..), you scan paramList and compare to dependencies dictionary for unknown dependency arguments. Request is a valid non-dependency parameter, so there needed to be a way to account for that when checking, which is why 'request' is mapped to a Request object in that dictionary, so an error doesn't get returned when checking Request object parameters in inner_handler. But instead of mapping those 2 objects in that dictionary, we could probably just get rid of those dictionary mappings and do something like

`if any(key not in specificDep and specificDep.get(key) is not Request and specificDep.get(key) is not Response for key in paramList):'

and handle everything within that function.